airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/schemas/conversions.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
Please add `"additionalProperties": true` to the root of all schemas newly added to support backward capability with all previous versions